parm/wflow/verify_det.yaml

```diff
@@ -21,6 +21,18 @@ default_task_verify_det: &default_task_verify_det
   queue: '&QUEUE_DEFAULT;'
   walltime: 00:30:00
 
+task_parse_vx_config_det:
+  <<: *default_task_verify_det
+  command: '&LOAD_MODULES_RUN_TASK_FP; "run_vx" "&JOBSdir;/JREGIONAL_PARSE_VX_CONFIG"'
```

Comment:
There is no `JREGIONAL_PARSE_VX_CONFIG` in the jobs directory.  This is causing the `parse_vx_config_det` task to fail.